src/frappe.json

```
@@ -1,40 +1,88 @@
 {
     "manifest": {
-        "author": "Isabelinc#9113",
-        "name": "Catppuccin Frappe",
-        "version": "1.3.0",
+        "author": "nyraki",
+        "name": "catppuccin frappé",
+        "version": "1.0.0",
```

Probably makes more sense to bump this to the next major version?
```suggestion
        "version": "2.0.0",
```